JavaErickHernandez/Day3/Maps.java

```diff
@@ -0,0 +1,93 @@
+package bootcamp;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Scanner;
+
+public class Maps {
+	static String numberOfInputs = "Enter the count of phone numbers: ";
+	static String tryAgainMessage = "Try Again";
+	static String tryAgainPhoneMessage = "Each phone number has exactly 8 digits without any leading zeros";
+	static Map<String, Integer> phoneNumbers = new HashMap<String, Integer>();
```

Comment:
When you variable is a Collection, it is good practice put name in singular and them the the type, example: tableList, nameSet, etc...